### PR TITLE
fix(request_information): fix empty string

### PR DESCRIPTION
**Pull request by XdpCs**
fix empty string #133

### CODE REVIEW

**Comment by XdpCs** [comment]
@microsoft-github-policy-service agree

File: request_information_test.go [hunk 1]
```diff
@@ -141,18 +141,6 @@ func TestItDoesNotSetEmptySelectQueryParameters(t *testing.T) {
 	assert.Equal(t, "http://localhost/me", resultUri.String())
 }
 
-func TestItDoesNotSetEmptySearchQueryParameters(t *testing.T) {
```
Comment (andrueastman) [inline request_information_test.go:144]: Any chance we could keep/modify the test and assert that the url string contains the query parameter with an empty value?

Comment (XdpCs) [reply request_information_test.go:144]: I think I can modify it.

Comment (XdpCs) [reply request_information_test.go]: I fix it in the latest commit.

**Review by rkodev — APPROVED** [review]

**Review by baywet — APPROVED** [review]
Thank you for the contribution!

**Pull request merged**